pkgs/top-level/all-packages.nix

```
@@ -21299,6 +21299,8 @@ in
 
   man-pages = callPackage ../data/documentation/man-pages { };
 
+  man-pages-posix = callPackages ../data/documentation/man-pages-posix { };
```

This does not evaluate otherwise:

```suggestion
  man-pages-posix = callPackage ../data/documentation/man-pages-posix { };
```

---

```
error: function `callPackages` was called on a *single* derivation "man-pages-posix-2017-a"; did you mean to use `callPackage` instead?
(use '--show-trace' to show detailed location information)
```

---

Thanks, asleep at the wheel when doing that rename apparently.